health-services/project-factory/src/server/utils/onGoingCampaignUpdateUtils.ts

```
@@ -0,0 +1,806 @@
+}
+
+function getCreatedResourceIds(resources: any, type: any) {
+  const processedType = type === 'boundary'
+    ? 'boundaryWithTarget'
+    : (type.includes('With') ? type.split('With')[0] : type); return resources
+      .filter((item: any) => item.type === processedType)
+      .map((item: any) => item.createResourceId);
+}
+
+function buildSearchCriteria(request: any, createdResourceId: any, type: any) {
+  let processedType = type === 'boundary'
```

Biome

[error] This let declares a variable that is only assigned once.

'processedType' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)